RecoCTPPS/TotemRPLocal/python/totemTimingLocalReconstruction_cff.py

```diff
@@ -10,3 +10,4 @@
     totemTimingRecHits *
     totemTimingLocalTracks
 )
+
```

Comment:
this looks unnecessary

Reply:
Indeed, leftovers from some vim hooks... will remove in a later commit/rebase!